server/controllers/middleware/unlockCookie.js

```diff
@@ -0,0 +1,13 @@
+const jwt = require('jsonwebtoken');
+
+exports.unlockCookie = (req, res, next) => {
+  jwt.verify(req.cookies.jwt, process.env.PRIVATEKEY, (err, unlockedCookie) => {
```

Comment:
there is another folder called utils , you should put your jwt verfiy logic on it